src/imsosorry/uwuification.py

```diff
@@ -107,12 +109,25 @@ def emoji_replace(match: re.Match, strength: float = 0.0) -> str:
     return match_string
 
 
-def uwuify(text: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1) -> str:
+def tildes(match: re.Match, strength: float = 0.0):
+    match_string = match.group()
+    if random.random() < strength:
+        return "~"
+    return match_string
+
+
+def tildify(text: str, strength: float) -> str:
+    """Adds some tildes to spaces."""
+    return REGEX_TILDE.sub(partial(tildes, strength=strength), text, 0)
+
+
+def uwuify(text: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1, tilde_strength: float = 0.1) -> str:
```

Comment:
This line is now too long for `black`
```suggestion
def uwuify(
    text: str,
    *,
    stutter_strength: float = 0.2,
    emoji_strength: float = 0.1,
    tilde_strength: float = 0.1,
) -> str:
```